src/parameterFactory.js

```diff
@@ -0,0 +1,124 @@
+import {
+  DateInput,
+  EmailInput,
+  NullableBooleanInput,
+  NumberInput,
+  TextInput,
+} from 'react-admin';
+import React from 'react';
+
+function guessType(parameter, fields) {
+  if (parameter.variable.match(/.*\[exists\]/i) !== null) {
```

Comment:
The `return` could be on the same line for consistency (IIRC, we do that evrywhere), same for similar one-line constructs